html-forms/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+  <html>
+  <head>
+    <title>html forms</title>
+  </head>
+  <body>
+    <h1>Coding form!</h1>
+    <form action="http://localhost:5500/html-forms/index.html">
+      <p>Email</p>
+      <input type="text">
+    </form>
+    <form action="http://localhost:5500/html-forms/index.html">
```

Comment:
You should only have 1 `form` element. Everything beneath your `h1` on line 7 should be wrapped into one `form` element.